lightning/src/onion_message/messenger.rs

```diff
@@ -1127,7 +1127,7 @@ where
 			.map_err(|_| SendError::PathNotFound)
 	}
 
-	fn create_blinded_path(&self) -> Result<BlindedPath, SendError> {
+	fn create_blinded_path(&self) -> Result<Vec<BlindedPath>, SendError> {
```

Comment:
No need to update this method if we aren't changing the behavior.

Reply:
Got it!
Reverted the onion_messenger.rs changes in [4461115](https://github.com/lightningdevkit/rust-lightning/commit/4461115e88f7fb3e0a366a24aba8a32bc6c7b408)
Thanks!